internal/config/moduleconfig/module_config.go

```diff
@@ -48,16 +55,109 @@ type TemplateConfig struct {
 	OutputDir  string `yaml:"outputDir"`
 }
 
+// A "nice" wrapper around findMissing()
+func (cfg ModuleConfig) collectMissing() []string {
+	var missing []string
+	findMissing(reflect.ValueOf(cfg), "", "", &missing)
+
+	return missing
+}
+
 func LoadModuleConfig(filePath string) (ModuleConfig, error) {
 	config := ModuleConfig{}
+
+	var required = []string{}
+	fmt.Printf("%+#v\n", required)
```

Comment:
I assume this is not needed anymore? since we're treating non-omitempty as required

Reply:
Yeah, for sure! Thanks.